### PR TITLE
Display config name and description in workflow node

### PR DESCRIPTION
**Pull request by brittanykondo**
Display config name and description in node:
Screenshot 2024-09-27 at 8 58 10 AM

### CODE REVIEW

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue [hunk 1]
```diff
@@ -120,3 +128,8 @@ watch(
 	}
 );
 </script>
+<style scoped>
+.description {
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue:132]:
```suggestion
h6 + p {
```

File: packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue [hunk 2]
```diff
@@ -1,6 +1,14 @@
 <template>
 	<section>
-		<tera-operator-placeholder :node="node" />
+		<div
+			v-if="node.state.transientModelConfig.id && node.state.transientModelConfig.id.length > 0"
+			class="pl-2 pr-2 pb-3"
+		>
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue:6]: Why `transientModelConfig.id` can have a length? Do you want use `isEmpty` from lodash instead?
Also with all that classes, you can just make a proper CSS selector like 
```css
div {
   padding-left: ....
  ...
  h6 {
      padding-bottom
  }

  h6 + p {
     text-subdued
  }
}
```

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue (hunk 2 above)
Comment (brittanykondo) [reply packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue]: Done!

**Pull request merged**